model : add grades model

zsabri91 opened the pull request:
Added a csv file, which columns of eid, grade , course, and assighnment ,id.

* added two fake entries in the csv
* didn't use student names because we think we will put the students, info in>
* use the course name but for assignments, we use the ID

issue #1

Commit:
model : add grades model

Added a csv file, which columns of eid, grade , course, and assighnment ,id.

* added two fake entries in the csv
* didn't use student names because we think we will put the students, info in>
* usethe course name but for assighnments, we use the I

Commit:
model :add student model

Added a HTML file which has a small message of the student name

* added a fake entries in the html
* I didn't add the student detalis because it is already in the grades.csv

issue #1

Ali-Almadhagi reviewed (changes requested):
you have a typo for the word "assignment" in grades.csv

Commit:
Fixed the Typo

This commit is because of my reviewr Ali almadhagi I had a tupo in the word assignment and I fixed it

* I fixed the my Typo
* I change the wrod assighnment to assignment

issue #1
